spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require 'pathname'
 require 'timeout'
+require 'active_support/core_ext/string/strip'
```

Comment:
Could you see this commit? c67dbaf
This change is needed to run RSpec with one file.